### PR TITLE
feat: add sections our team

### LINKED ISSUE #18: feat: add sections our team
### Describe the feature you'd like to request

Existing:
image

Mockup:
image

### Describe the solution you'd like

-

### Describe alternatives you've considered

-

### PR DESCRIPTION
**Pull request by Nunuyy**
Added a solutions section that explains what solutions are available at Sawala tech

## Changes
- Add Sections OurTeam #18 

## Screenshoots

- Sections OurTeam

- Responsive

| Mobile            | Tablet            |
| ------------- | ------------- |
|  |  |

## Checklist
- [ ] Copywriting 
- [ ] Requires dependency update?
- [ ] Automated tests
- [ ] Looks good on large screens
- [ ] Looks good on mobile
- [ ] Related issues linked using fixes #18

### CODE REVIEW

File: src/components/Card/Founder/index.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import { FC, useEffect, useState } from 'react'
+import tw, { styled } from 'twin.macro'
+
+const ContentFounder = styled.div`
+  ${tw`place-items-center items-center pl-5 my-auto flex w-full h-[220px] cursor-default transition ease-in-out delay-100 focus:outline-none hover:scale-110 hover:bg-primary hover:bg-opacity-10 duration-200 bg-gray-50 rounded-xl`}
```
Comment (wanfahdiva) [inline src/components/Card/Founder/index.tsx:7]:
```suggestion
  ${tw`place-items-center items-center pl-5 my-auto flex w-full h-[170px] md:h-[220px] cursor-default transition ease-in-out delay-100 focus:outline-none hover:scale-110 hover:bg-primary hover:bg-opacity-10 duration-200 bg-gray-50 rounded-xl`}
```

File: src/components/Card/Founder/index.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+`
+const Title = styled.div`
+  ${tw`flex-1 pr-2 text-xl font-bold text-left text-gray-900 capitalize pl-7`}
+`
+const SubTitle = styled.div`
+  ${tw`text-base font-medium text-gray-500 pb-7`}
```
Comment (wanfahdiva) [inline src/components/Card/Founder/index.tsx:13]:
```suggestion
  ${tw`text-base font-medium text-gray-500 pb-5 md:pb-7`}
```

File: src/components/Card/Team/index.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import tw, { styled } from 'twin.macro'
+import { FC, useEffect, useState } from 'react'
+
+const ContentTeam = styled.div`
+  ${tw`pl-5 flex w-full h-[220px] cursor-default transition ease-in-out delay-100 focus:outline-none hover:scale-110 hover:bg-primary hover:bg-opacity-10 duration-200 bg-gray-50 rounded-xl place-items-center`}
```
Comment (wanfahdiva) [inline src/components/Card/Team/index.tsx:7]:
```suggestion
  ${tw`pl-5 flex w-full h-[170px] md:h-[220px] cursor-default transition ease-in-out delay-100 focus:outline-none hover:scale-110 hover:bg-primary hover:bg-opacity-10 duration-200 bg-gray-50 rounded-xl place-items-center`}
```

File: src/components/Card/Team/index.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+`
+const Title = styled.div`
+  ${tw`flex-1 text-xl font-bold text-left text-gray-900 capitalize pl-7`}
+`
+const SubTitle = styled.div`
+  ${tw`text-base font-medium text-gray-500 pb-7`}
```
Comment (wanfahdiva) [inline src/components/Card/Team/index.tsx:13]:
```suggestion
  ${tw`text-base font-medium text-gray-500 pb-5 md:pb-7`}
```

**Review by wanfahdiva — CHANGES_REQUESTED** [review]

**Comment by sutisnamulyana** [comment]
- [ ] Typo: Lead Develover -->Lead Developer
- [ ] Semua tidak pake gelar academic
- [ ] Link linkedin dibikin new tab

**Review by sutisnamulyana — APPROVED** [review]
Good to go

**Pull request closed without merging**